src/posts/xrisk.md

```diff
@@ -128,33 +121,23 @@ This is an AI, using the aforementioned AutoGPT + GPT-4, that is instructed to "
 When it was turned on, it autonomously searched the internet for the most destructive weapon and found the [Tsar Bomba](https://en.wikipedia.org/wiki/Tsar_Bomba), a 50 megaton nuclear bomb.
 It then posted a tweet about it.
 Seeing an AI reason about how it will end humanity is both a little funny and terrifying.
-ChaotGPT luckily didn't get very far in its quest for dominance.
-The only reason it didn't get very far: it wasn't that smart.
-But what would happen if it was extremely intelligent?
-What could have happened if it was smart enough to [hack into nuclear silos](https://t.co/UNPERx9NdL)?
+Luckily ChaosGPT didn't get very far in its quest for dominance.
+The reason it didn't get very far: it wasn't that smart.
+As language models improve the threat from AIs such as ChaosGPT will increase.
 
-The dangerous capacity is intelligence.
-There is no need for consciousness, a body, or any underlying motivation.
-It does not matter if it is just a chatbot - it can still be an existential threat with the wrong prompt, a tool like AutoGPT and sufficient intelligence.
-If an AI is smart enough to take over, at some point in time it will try to do this, simply because people will try all sorts of things.
```

Comment:
This argument feels too important to leave out. I even think it deservers an entire paragraph, instead of one sentence.

Every time we use an AI, we create a new instance. Every prompt is different, and every prompt means that instance could land in a configuration where it decides that spreading itself is sensible. 

It only has to go wrong once. We can have 99999 AI's that are friendly oracles that answer our questions and do their tasks, and only one instance that decides that it should seek power, and we're fucked.

Reply:
Sure, I think adding a paragraph on this could be good. But I think the way it is currently stated is hard to understand.

Reply:
Fair